ObjAndFunc.js

```diff
@@ -0,0 +1,57 @@
+// Создать объект car, добавить к нему свойство color со значением 'черный'
+//Изменить свойство color объекта car на 'зеленый'
+//В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
+let car={
+    color:"черный",
+    power(){
```

Comment:
let car={
    color:"черный",
    power: 500, 
    } 
};
So you can use power, when outputting to the console, it will be impossible to use properties further.